### DISCUSSION TITLE
A request to become a maintainer

### DISCUSSION THREAD

**Discussion by mobin-2008** [davmac314/dinit#83] [General]
Hello.
I think this is the first request to become a Dinit project maintainer by a user (maybe not!).
I request to become one of the maintainers of the Dinit project. The most important thing I have in mind that is possible with maintainer access:
1. Reviewing the use of labels in issues and PRs. Currently, the project does not use labels.

I know other duties of a maintainer:
1. Review of issues and PRs
2. Issuance of access to CI implementation for new PRs
And...

Maybe this request seems bad and "impudent" but it really is not.
Dinit is the only init that does the job of an init properly: a compact program that runs as PID 1 and is responsible for controlling system services. None of the inits I tried produced the desired result (SystemD, sysvinit, OpenRC, runit).

**Comment by davmac314** [comment]
I've got no problem in principle (so I've sent an invite) but I'm not sure there is much to do as a collaborator at this stage. True I don't use labels for issues, but there are only a handful of issues open so this has never been a problem. Well, we'll see!
